### PR TITLE
Automated test #1

### LINKED ISSUE #13: Automated test #1
The user can create a new comment when filling all the required fields

### PR DESCRIPTION
**Pull request by oteplova**
Closes #13

### CODE REVIEW

File: src/main/java/net/comments/Test1.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class Test1 {
+    private final WebDriver[] driver = new WebDriver[1];
+
+    @Test
+    public void test() {
+        final String page = this.driver().getPageSource();
```
Comment (extsoft) [inline src/main/java/net/comments/Test1.java]: It's bug if you load the page content before even page is open (line 21). That's why assertion on line 29 won't work correctly.

File: src/main/java/net/comments/Test1.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        String commentsText = "Test text";
```
Comment (extsoft) [inline src/main/java/net/comments/Test1.java]: Make `final`.

File: src/main/java/net/comments/Test1.java [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+        String number = "31";
```
Comment (extsoft) [inline src/main/java/net/comments/Test1.java]: Make `final`.

File: src/main/java/net/comments/Test1.java [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+        String savedNumber, savedCommentsText, inactivationSign, categoryName;
```
Comment (extsoft) [inline src/main/java/net/comments/Test1.java]: Don't use batch assignment. Create a variable when you are going to use it. For instance, in line 31.

File: src/main/java/net/comments/Test1.java [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+        driver().findElement(By.xpath("//*[@value=\">\"]")).click();
+        driver().findElement(By.xpath("//*[@value=\"Save & Return\"]")).click();
+        driver().findElement(By.linkText("4")).click();
+        MatcherAssert.assertThat("Comment is not present on the page", !page.contains(commentsText));
+
+        savedNumber = driver().findElement(By.xpath("//*[@id = 'main']//tr[1]//td[2]")).getText();
+        MatcherAssert.assertThat("Number is not " + number, savedNumber.contains(number));
+
+        savedCommentsText = driver().findElement(By.xpath("//*[@id = 'main']//tr[1]//td[3]")).getText();
+        MatcherAssert.assertThat("Comments text does not contain " + savedCommentsText, savedCommentsText.contains(savedCommentsText));
```
Comment (extsoft) [inline src/main/java/net/comments/Test1.java]: Actual and Expected are the same.

**Review by extsoft — CHANGES_REQUESTED** [review]
@oteplova please solve the comments.

**Comment by oteplova** [comment]
@extsoft I've fixed comments. Please check

**Review by extsoft — APPROVED** [review]
@oteplova great!

**Pull request merged**